src/plugin/Plugin.h

```diff
@@ -855,7 +880,10 @@ class Plugin {
 	 *
 	 * args: A list of the hooks arguments.
 	 */
-	virtual void MetaHookPre(HookType hook, const HookArgumentList& args);
+	// TODO: unfortunately deprecated virtual methods don't flag when you override them
```

Comment:
Yeah, sucks, but think the only TODO is to document the deprecation in `NEWS`.  Could even say that having compiler emit warnings is only a convenience but not a substitute for expecting users to read release notes as the official way of being told about upcoming deprecations.